PendingReleaseNotes

```diff
@@ -9,6 +9,10 @@
   Admins can no longer enable the older, deprecated landing page layout by
   adjusting FEATURE_TOGGLE_DASHBOARD.
 * CephFS: The `peer_add` command is deprecated in favor of the `peer_bootstrap` command.
+* RADOS: When objects data is read during deep scrubs, the data is read in strides
+  (with delays to the scrubbing process between each read, in order to avoid monopolizing
+  the I/O capacity of the OSD.)
+  The default stride size (`osd_deep_scrub_stride`) was 512 KBytes, and iss now modified to be 4 MBytes.
```

Comment:
Suggest :confval: instead of the parens

and is now 4 MiB.

Reply:
@anthonyeleven - thanks.
:confval: ? I did not find a configuration value surrounded by colons in teh doc. Did you mean double back-quotes?

Reply:
I meant like this example in git/ceph/doc/radosgw/config-ref.rst:

The older and now non-default``civetweb`` frontend has a threading model that uses a thread per
connection and hence is automatically throttled by :confval:`rgw_thread_pool_size`
when accepting connections. The newer and default ``beast`` frontend is
not limited by the thread pool size when it comes to accepting new

But since this is PendingReleaseNotes not an RST doc, ignore that.  "iss" should still be "is" though.